src/ApprenticePortal/SFA.DAS.ApprenticePortal/InnerApi/ApprenticeAccounts/Requests/GetMyApprenticeshipRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+using System;
+
+namespace SFA.DAS.ApprenticePortal.InnerApi.ApprenticeAccounts.Requests
+{
+    public class GetMyApprenticeshipRequest : IGetApiRequest
+    {
+        private readonly Guid _id;
+
+        public GetMyApprenticeshipRequest(Guid id)
+        {
+            _id = id;
+        }
+
+        public string GetUrl => $"apprentices/{_id}/my-apprenticeship";
```

Comment:
Should be "/myapprenticeship" and not "my-apprenticeship" on the call to Apprentice Accounts Inner API.